src/apollo/resolvers/post.ts

```diff
@@ -132,7 +126,7 @@ const postResolvers = {
       try {
         await saveImages(post, images);
       } catch (e) {
```

Comment:
Should we remove the `(e)`s that aren't being used?

Reply:
Well originally we sent that back with the ApolloError. We have to an argument for the catch block or it won't compile. Should we find a way to log errors internally?

Reply:
This will be completed in a new PR!